src/main/java/pl/danyboy/Game.java

```diff
@@ -5,22 +5,31 @@
 public class Game {
     private final PasswordManager passwordManager;
     private final GuessStage guessStage;
+    private final LineReader lineReader;
+    private final PasswordStage passwordStage;
 
-    public Game(PasswordManager passwordManager, GuessStage guessStage) {
+    public Game(PasswordManager passwordManager, GuessStage guessStage, LineReader lineReader, PasswordStage passwordStage) {
         this.passwordManager = passwordManager;
         this.guessStage = guessStage;
+
+        this.lineReader = lineReader;
+        this.passwordStage = passwordStage;
     }
 
     public void roundsOfGame(List<Player> players) throws IllegalStateException {
         int rounds = 4;
         for (int i = 0; i < rounds; i++) {
             System.out.println("Rozpoczęła się runda " + (i + 1));
             String randomPassword = passwordManager.getRandomPassword();
-            System.out.println(randomPassword);
+            System.out.println(passwordStage.getObscuredPassword(randomPassword, null));
```

Comment:
Przekazywanie nulli jako parametr to zła praktyka w logice programu (w testach jednostkowych jeszcze ujdzie), może lepiej po prostu zrobić inną metodę? Albo choć pustego stringa przesłać.